saleor/settings.py

```diff
@@ -10,6 +10,7 @@
 from django_prices.utils.formatting import get_currency_fraction
 from sentry_sdk.integrations.django import DjangoIntegration
 
```

Comment:
Remove this new line.

Reply:
This was not altered. Should the line be removed anyway?

Reply:
It wasn't altered because you didn't add the import right after the last one.

Reply:
Whoops 😅